pkg/config/gui-server/config.go

```diff
@@ -22,6 +24,13 @@ var _ config.Config = &GuiServerConfig{}
 
 func DefaultGuiServerConfig() *GuiServerConfig {
 	return &GuiServerConfig{
-		Port: 5683,
+		Port:      5683,
+		GuiConfig: &GuiConfig{},
 	}
 }
+
+// Not yet exposed via YAML and env vars on purpose. All of those are autoconfigured
+type GuiConfig struct {
+	ApiUrl      string
+	Environment string
```

Comment:
Why is GUI has to be statically configured with `Environment` instead of fetching this info via REST API ?

Reply:
Because there is no REST API for config yet.
https://github.com/Kong/kuma/issues/429 I don't want to start implementing this right now. Once we have it, we can move fetching the environment from here